plugins/Convert_Masked.py

```diff
@@ -8,11 +8,13 @@
 class Convert():
     def __init__(self, encoder, blur_size=2, seamless_clone=False, mask_type="facehullandrect", erosion_kernel_size=None, **kwargs):
         self.encoder = encoder
-
         self.erosion_kernel = None
+        self.erosion_kernel_size = erosion_kernel_size       
         if erosion_kernel_size is not None:
-            self.erosion_kernel = cv2.getStructuringElement(cv2.MORPH_ELLIPSE,(erosion_kernel_size,erosion_kernel_size))
-
+            if erosion_kernel_size > 0:
+                self.erosion_kernel = cv2.getStructuringElement(cv2.MORPH_ELLIPSE,(erosion_kernel_size,erosion_kernel_size))
+            elif erosion_kernel_size < 0:

```

Comment:
Could there be a potential issue if `erosion_kernel_size = 0`?

Reply:
No @leftler the default for the variable is None. Before this commit if you passed -e = 0 it would cause an OpenCV error because it needs a positive number. In this commit erosion_kernel_size 0 just keeps erosion_kernel as None and nothing happens.